base/sparse/cholmod.jl

```diff
@@ -1248,7 +1248,14 @@ end
         Hermitian{Complex{<:Real},SparseMatrixCSC{Complex{<:Real},SuiteSparse_long}}};
         shift = 0.0) -> CHOLMOD.Factor
 
-Compute the LDLt factorization of `A`, reusing the symbolic factorization `F`.
+Compute the ``LDL'`` factorization of `A`, reusing the symbolic factorization `F`.
```

Comment:
This is `cholfact!`, not `ldltfact`. Something is wrong here. Maybe I've made a copy-paste error when moving the documentation over here.